dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/step/validation/MetadataValidation.java

```diff
@@ -102,6 +105,28 @@ public List<ErrorRest> validate(SubmissionService submissionService, InProgressS
         return getErrors();
     }
 
+    private boolean isValidComplexDefinitionMetadata(DCInput input, List<MetadataValue> mdv) {
+        if (input.getInputType().equals("complex")) {
+            int complexDefinitionIndex = 0;
+            for (String complexDefinitionInputName : input.getComplexDefinition().getInputs().keySet()) {
+                Map<String, String> complexDefinitionInputValues =
+                        input.getComplexDefinition().getInputs().get(complexDefinitionInputName);
+
+                List<String> filledInputValues = null;
+                if (("true").equals(complexDefinitionInputValues.get("required"))) {
+                    filledInputValues = new ArrayList<>(Arrays.asList(
+                            mdv.get(0).getValue().split(DCInput.ComplexDefinitions.getSeparator(),-1)));
```

Comment:
mdv empty?

Reply:
the mdv won't be empty here because above is a condition to check if mdv is not empty